docker/README.md

```diff
@@ -13,7 +13,7 @@ Bookkeeper needs [Zookeeper](https://zookeeper.apache.org/) in order to preserve
 Just like running a BookKeeper cluster in one machine(http://bookkeeper.apache.org/docs/latest/getting-started/run-locally/), you can run a standalone BookKeeper in one docker container, the command is:
 ```
 docker run -it \
-     --env JAVA_HOME=/usr/lib/jvm/jre-1.8.0 \
+     --env JAVA_HOME=/usr/lib/jvm/jdk-11 \
```

Comment:
Should we install the JRE instead? I suppose installing JDK instead of JRE would increase the image size. I'm not sure whether that increase in size is significant in relation to the overall image size or not. What are your thoughts about this?

Reply:
I would like, but in current 4.11 we are using the presence of shell to detect jdk11 and it is not present in the.

We should fix that problem, then we can use the jre.

Btw I don't know a good way to detect jdk11+

Reply:
@ravisharda here it is a fix to support JRE and only only JDK
https://github.com/apache/bookkeeper/pull/2436